### PR TITLE
makes captain have level 1 surgery

### PR DESCRIPTION
**Pull request by Anystar1**
## About The Pull Request

<s>makes captain have level 3 surgery</s> 
did this PR as a way to see how a PR is made but sent it to the wrong place so im editing this PR instead
makes captain have level 1 surgery
## Why It's Good For The Game

so captain can do surgery when theres no shipside medical staff
## Changelog
:cl:
add: Captain now has level 1 surgery.
/:cl:

### CODE REVIEW

**Comment by Hekzder** [comment]
It's called using the autodoc. Skill issue. Captain doesn't need to know how to do literally everything.

**Comment by Anystar1** [comment]
wrong place

**Comment by Anystar1** [comment]
edited the PR

File: code/datums/skills.dm [hunk 1]
```diff
@@ -322,6 +322,7 @@ engineer, construction, leadership, medical, surgery, pilot, police, powerloader
 	leadership = SKILL_LEAD_MASTER
 	police = SKILL_POLICE_MP
 	medical = SKILL_MEDICAL_COMPETENT
+	surgery = SKILL_SURGERY_PROFESSIONAL
```
Comment (BraveMole) [inline code/datums/skills.dm:325]: This is not level 1 ???

**Review by BraveMole — CHANGES_REQUESTED** [review]

**Comment by Pariah919** [comment]
give him the skill to use an autodoc without fumbling

**Comment by Nirvan4** [comment]
This is not bad to captain, I had plenty of experiences as cap with no crew, its really painful to do everything and dont have skills to do half of them.

File: code/datums/skills.dm (hunk 1 above)
Comment (Anystar1) [reply code/datums/skills.dm]: i think i made it amateur now?

**Review by BraveMole — APPROVED** [review]

**Pull request merged**